app/models/ocelot/Stanza.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.ocelot
+
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+
+sealed trait Stanza
+
+
+/**
+ * Define Callout Stanza
+ */
+case class CalloutStanza(
+                        noteType: String,
```

Comment:
consider making `noteType` a type (e.g. `sealed trait CalloutType`) allowing matching on the callout types more specific than a text match.